### PR TITLE
Staking key chain for HD wallets

### PR DESCRIPTION
**Pull request by SebastienGllmt**
As per our discussion during the CIP call, I split up https://github.com/cardano-foundation/CIPs/pull/33 into two CIPs. This CIP contains the description of `chain=2` whereas CIP1852 now outlines the derivation strategy and acts as a registry of chain types.

### CODE REVIEW

File: StakingKey/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+m / purpose' / coin_type' / account' / chain / address_index
+```
+
+We set `chain=2` to indicate the *staking key chain*. Keys in this chain MUST follow the accounting model for transactions and SHOULD be used for *reward addresses*
+
+### *address_index* value
+
+We RECOMMEND wallets only use `address_index=0` for compatibility with existing software. This also avoids the need for staking key discovery.
+
+Wallets that use multiple staking keys are REQUIRED to use sequential indexing with no gaps. This is to make detection of mangle addresses (addresses where the payment key belongs to the user, but the staking key doesn't) easier.
```
Comment (KtorZ) [inline StakingKey/README.md:51]: *the detection of mangle addresses **possible** (

I imagine.

**Review by KtorZ — APPROVED** [review]

**Comment by crptmppt** [comment]
can we change the structure to match the new repo format (so Github can display appropriately)?

File: CIP-0011/CIP-0011.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Comments-URI: https://forum.cardano.org/t/staking-key-chain-for-hd-wallets/41857
+Status: Draft
+Type: Standards
+Created: 2020-11-04
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Starting with the Shelley hardfork, Cardano makes use of both the *UTXO model* and the *account model*. To support both transaction models from the same master key, we allocate a new chain for [CIP1852](../CIP1852)
```
Comment (crptmppt) [inline CIP-0011/CIP-0011.md:14]: need to change link to `../CIP-1852`

File: CIP-0011/CIP-0011.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Blockchains like Ethereum does not use the UTXO model and instead uses the [*Account model*](https://github.com/ethereum/wiki/wiki/Design-Rationale#accounts-and-not-utxos) for transactions.
+
+## Motivation
+
+Generally it's best to only use a cryptographic key for a single purpose, and so it's best to make the staking key be separate from any key used for UTXO addresses.
+
+## Specification
+
+Recall that [CIP1852](../CIP1852) specifies the following derivation path
```
Comment (crptmppt) [inline CIP-0011/CIP-0011.md:39]: need to change link to ../CIP-1852

**Review by crptmppt — CHANGES_REQUESTED** [review]

**Review by crptmppt — APPROVED** [review]
will add pr for link fixes

**Pull request merged**